nautobot/ui/src/views/generic/GenericView.js

```diff
@@ -120,23 +121,28 @@ export default function GenericView({
             width="full"
         >
             <Navbar appState={currentState} />
-            <Box flex="1" overflow="auto">
-                <Breadcrumbs paddingX="md">
-                    {breadcrumbs.map((props) => (
-                        <Breadcrumb {...props} />
-                    ))}
-                </Breadcrumbs>
 
-                <NautobotGrid
-                    alignItems="start"
-                    columns={columns}
-                    rows={rows}
-                    background={gridBackground}
-                    gridAutoRows="auto"
-                >
-                    {children}
-                </NautobotGrid>
-            </Box>
+            <Flex flex="1" overflow="hidden">
+                <Box flex="1" overflow="auto">
+                    <Breadcrumbs paddingX="md">
+                        {breadcrumbs.map((props) => (
+                            <Breadcrumb {...props} />
+                        ))}
+                    </Breadcrumbs>
+
+                    <NautobotGrid
+                        alignItems="start"
+                        columns={columns}
+                        rows={rows}
+                        background={gridBackground}
+                        gridAutoRows="auto"
+                    >
+                        {children}
+                    </NautobotGrid>
+                </Box>
+
+                <FiltersPanelContainer />
```

Comment:
Similarly, shouldn't this be only in `ObjectList.js` rather than in `GenericView.js`?

Reply:
`ObjectList` view is lower in the tree structure, which rules out the possibility of filters panel to be a part of it. How I see it - currently the layout consists of these 4 fundamental elements: header, sidebar, main content and filters panel. So if we want to shuffle them, we need to change the root layout. Perhaps it would be a good idea, since you said filters panel is going to be used only on object list view anyway, but at the same time for now it is designed to be a top level layout element and if that needs to change, then let's do it outside of scope of this PR